sites/page-details/_screen-md-min.scss

```diff
@@ -0,0 +1,16 @@
+/*
+ * Share/feedback-specific overrides (medium view and over)
+ */
+
+// Add a larger top margin on the share button (only if its plugin has initialized) if adjacent to the page feedback widget.
+.pagedetails {
+	div {
+		&.gc-pg-hlpfl {
+			+ {
+				.wb-share-inited {
+					margin-top: 21px;
```

Comment:
I'm not a fan of "arbitrarily" assigning pixels to center, but I understand the necessity of this to get both sides to be vertically centered.

Maybe in the future we could have a small Javascript to calculate where the center is and assign the proper margins... and keep the values you set as a fallback. Thinking of Javascript, just brainstorming here, would a mix of [wb-eqht Javascript](https://wet-boew.github.io/wet-boew/demos/equalheight/equalheight-en.html) and [align-self: center; from my WET PR](https://github.com/wet-boew/wet-boew/pull/9428/files#diff-fc38b9e10cf36e1f8aed3186489255e65952ff64bbe5c1f20a2ea5fbf16f01b6R56-R58) do the trick?

Another alternative is having `margin-top: 0px;` to align both sides to the top instead of "arbitrarily" assigning pixels to center, but maybe it isn't as aesthetically pleasing.
<img width="760" alt="Screen Shot 2022-09-28 at 12 00 51 AM" src="https://user-images.githubusercontent.com/4353199/192684950-c986b3bd-4d98-4bc2-be32-9e8860afa09a.png">
(This comment applies to all instances of margin-top being assinged to the Share this page button in this PR.)

Reply:
IMO it should be feasible to use flexbox CSS to vertically-center both the widget and the share button. But there's a catch... :(.

The reason I didn't go with that approach in the first place is because interacting with the widget can cause its height to change. For example, if a user picks no, the widget transitions to a much-taller questionnaire. That would cause the share button to shift around if it was vertically-centered. So with that in mind, I felt like the best compromise approach was to hardcode a specific top margin.

If you really want to go all out, I'd suggest using a mix of CSS and JS for the best of both worlds:
1. Hardcode a CSS class on the share button to vertically-center it when adjacent to the widget in small view and up.
   * That's the same solution I discounted above :P
   * This method would be much more responsive than JS
2. Use a JS plugin to detect clicks on the widget's yes/no buttons and do the following **before** the do action plugin kicks in:
  a. Capture the share button's vertical position
  b. Inject a ``style`` attribute onto the share button to set its offset
  c. Remove step 1's CSS class
  d. Update the offset if the view mode changes

Doing all that might not be worth the effort though.

IMO the only scenario that all-out approach would benefit vs the status quo would be in browsers that use very heavy text-only zooming. Or to future-proof GCWeb in case the number of lines in the widget's content changes in the future. The widget's French and English content currently use the same number of lines in small view and up.

Reply:
Alright so I tried different things, even got to try using the JS equal height plugin to ease styling, but it didn't get me anywhere. I thought maybe the JS equal height plugin could assign an equal height value on both sides on load and then "forget" about it once the No button is clicked and left side expand, but that didn't work for me as I expected.

So, it's either we stabilize it with the "Share this page" button aligned to the top or write up a small Javascript plugin to style it (as you proposed). 

Reason I came to this conclusion is because we need to apply the `gc-pg-hlpfl` inside its parent div with columns, as such: 
<img width="951" alt="Screen Shot 2022-10-13 at 6 23 56 PM" src="https://user-images.githubusercontent.com/4353199/195729920-2bfb5811-c325-41a2-83f8-d2e2ab361382.png">

This is necessary to be able to handle both the RAP and OPUFT inside the same columns and through a data-ajax call as seen here: https://wet-boew.github.io/GCWeb/sites/feedback/feedback-en.html

Reply:
I was able to resolve this by using the ``has()`` pseudo-class (lol) and verified it in Chrome/Edge/Firefox Dev Edition.

[According to caniuse.com's ``has() page``](https://caniuse.com/css-has), WebKit first added support for it in December 2021 and Blink caught-up in April. All the major browsers that use those engines (i.e. Chrome/Edge/Safari) currently have it enabled by default. In terms of WebKit/Blink reskin browsers, Opera has already added it and I presume others will catch up over the next few months.

As for Gecko/Firefox, they currently have experimental support for ``has()`` that can be activated by setting ``layout.css.has-selector.enabled`` to ``true`` in ``about:config``. There's no ETA for when it'll be enabled by default, but I presume it's just a matter of time given that Mozilla probably won't want to fall too far behind their competitors for such a major feature.

Btw [WebKit's blog has a nice background story](https://webkit.org/blog/13096/css-has-pseudo-class/) about the history of ``has()`` and how it recently started gaining real browser support after being stale for ages.